rg_instructor_analytics/run_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PROJECT_ROOT=$(pwd)
+export SERVICE_VARIANT="lms"
+export DISABLE_MIGRATIONS="false"
+cd ~/edx-platform/
+
+MYSQL=$(which mysql)
+GRANT_REQUEST1="GRANT ALL PRIVILEGES ON test_edxapp.* TO 'edxapp001'@'localhost'"
+GRANT_REQUEST2="GRANT ALL PRIVILEGES ON test_edxapp.* TO 'migrate'@'localhost'"
+GRANT_REQUEST3="GRANT ALL PRIVILEGES ON test_edxapp_csmh.* TO 'edxapp001'@'localhost'"
+GRANT_REQUEST4="GRANT ALL PRIVILEGES ON test_edxapp_csmh.* TO 'migrate'@'localhost'"
+$MYSQL -u root mysql -e "$GRANT_REQUEST1"
+$MYSQL -u root mysql -e "$GRANT_REQUEST2"
+$MYSQL -u root mysql -e "$GRANT_REQUEST3"
+$MYSQL -u root mysql -e "$GRANT_REQUEST4"
+
+pytest $PROJECT_ROOT/tests -c $PROJECT_ROOT/pytest.ini
+
+#paver test_bokchoy -t openedx/features/edx_rg_instructor_analytics/rg_instructor_analytics/test/ --disable_migrations
```

Comment:
Do we need this commented line?

Reply:
No, it is the artifact, and it will be deleted in next commit.